users/migrations/0002_alter_team_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.9 on 2021-12-01 18:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='team',
+            name='name',
+            field=models.CharField(default=None, max_length=255, unique=True),
+        ),
```

Comment:
why do you write it in new file and don't add unique=True to 001_initial file?
 operations = [
        migrations.CreateModel(
            name='Team',
            fields=[
                ('id', models.BigAutoField(auto_created=True,
                 primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=255,
                 blank=False, default=None)),
                ('description', models.CharField(max_length=500, unique=True)),
            ],
        ),

Reply:
This file was auto-generated by django after changing the original Team model.

Reply:
and is it possible to add there a field?